.github/buildkit.toml

```diff
@@ -0,0 +1,2 @@
+[worker.oci]
+  max-parallelism = 4
```

Comment:
Any improvements if we fix parallelism?

Reply:
The parallelism goes so big it sometimes runs out of memory without it if I remember correctly. Also faster